### DISCUSSION TITLE
probleme driver ASIO

### DISCUSSION THREAD

**Discussion by BenClam** [GrandOrgue/grandorgue#1213] [Developers]
Sur PC Intel i7 le driver ASIO Version 15 bien installé ne donne pas de son 
Comment faire ?

**Comment by oleg68** [comment]
1. Est'ce qui'l y a le son en autres applicatoones avec ASIO?
2. Si oui, postez ici la copie d'ecran de File->Settings->Audio. C'est possible, qui on doive changer l'util audio dans cet ecran pour l'utiliser.
